main.go

```diff
@@ -17,10 +17,10 @@ import (
 	"github.com/alexflint/go-arg"
 
 	"github.com/knq/dburl"
-	"github.com/codercom/xo/internal"
-	"github.com/codercom/xo/models"
+	"github.com/knq/xo/internal"
```

Comment:
The imports got switched around again, this branch doesn't build without a search / replace.

Reply:
you should add this as a remote to the original package (`$GOPATH/src/github.com/knq/xo`)

Reply:
But, yes, in order to get this to work and to maintain compatibility with `origin` (`knq/xo`), I added our fork as a remote to my `github.com/knq/xo` package.

Reply:
This is the worst dependency ever.

:stuck_out_tongue_winking_eye: